components/skus/browser/skus_service_impl.cc

```diff
@@ -32,8 +32,15 @@ void OnFetchOrderCredentials(
     skus::FetchOrderCredentialsCallbackState* callback_state,
     skus::SkusResult result) {
   if (callback_state->cb) {
-    std::move(callback_state->cb).Run("");
+    const int error_code = static_cast<int>(result);
+    std::string error_message;
+    if (error_code != 0) {
```

Comment:
`skus::SkusResult` should be a C++ enum, I think this would be better as `if (error_code != skus::SkusResult::Ok)`